general/clients/index.md

```diff
@@ -12,6 +12,25 @@ Clients connect your devices to your Jellyfin server and let you view your conte
 
 Do you have a client that interfaces with Jellyfin and want to see it listed here? Please [submit a pull request](https://github.com/jellyfin/jellyfin-docs)!
 
+## Browsers
+
+Our goal is to provide support for these browsers. 
+
+ - last 2 Firefox versions
+ - last 2 Chrome versions
+ - last 2 ChromeAndroid versions
+ - last 2 Safari versions
+ - last 2 iOS versions
+ - last 2 Edge versions
+ - Chrome 27
+ - Chrome 38
+ - Chrome 47
+ - Chrome 53
+ - Chrome 56
+ - Chrome 63
```

Comment:
All these specific Chrome versions are basically for Tizen and WebOS, not really for desktops (though it should still work)

Reply:
If you know which go with which, i can add them in parenthesis

Reply:
We should make a note that we aren't opposed to supporting older browsers through polyfills and such, but it's not a guarantee.

Reply:
added

Reply:
We should probably remove all the Chrome versions and move "Older browsers may be supported, but are not guaranteed." after the list, as something like this:

```markdown
Older browsers may be supported as a result of the needs of specific web-based clients, but full functionality is not guaranteed on their desktop version.
```